tests/utils/test_general.py

```diff
@@ -57,23 +57,26 @@ def test_timestamp_to_datetime_invalid(input_dt):
     assert 'The timestamp "{0}" is an invalid format'.format(input_dt) == str(exc_info.value)
 
 
-@pytest.mark.parametrize('uid,label,backward,expected', [
-    ('123456', 'BugzillaBug', False, 'Commits that resolved Bugzilla bug 123456'),
-    ('123456', 'DistGitCommit', False, 'Builds built by commit 123456'),
-    ('123456', 'DistGitCommit', True, 'Bugzilla bugs resolved by commit 123456'),
-    ('123456', 'KojiBuild', False, 'Advisories attached to build 123456'),
-    ('123456', 'KojiBuild', True, 'Commits that built build 123456'),
-    ('123456', 'Advisory', False, 'Freshmaker events triggered by advisory 123456'),
-    ('123456', 'Advisory', True, 'Builds attached to advisory 123456'),
-    ('123456', 'FreshmakerEvent', False, 'Container builds triggered by Freshmaker event 123456'),
-    ('123456', 'FreshmakerEvent', True, 'Advisories that triggered Freshmaker event 123456'),
-    ('123456', 'ContainerKojiBuild', False, 'Container advisories attached to container build'
-        ' 123456'),
-    ('123456', 'ContainerKojiBuild', True, 'Freshmaker events that triggered container build'
-        ' 123456'),
-    ('123456', 'ContainerAdvisory', True, 'Container builds attached to container advisory 123456')
+@pytest.mark.parametrize('display_name,label,backward,expected', [
+    ('RHBZ#12345', 'BugzillaBug', False, 'Commits that resolved RHBZ#12345'),
+    ('commit #1234567', 'DistGitCommit', False, 'Builds built by commit #1234567'),
+    ('commit #1234567', 'DistGitCommit', True, 'Bugzilla bugs resolved by commit #1234567'),
+    ('python-requests-1.10-3', 'KojiBuild', False, 'Advisories attached to python-requests-1.10-3'),
```

Comment:
This text isn't quite right. It should be something like `Advisories that contain python-requests-1.10-3`.